docs/pages/includes/determinetls.mdx

```diff
@@ -0,0 +1,26 @@
+<Details title="How to tell if TLS Routing is Enabled" opened={false}>
+
```

Comment:
To prevent the user from leaving the page, I think it would be useful to add a one-sentence definition of TLS Routing here. This way, we can remove the link to the TLS Routing page later in these instructions.

Reply:
addressed